pages/user.js

```diff
@@ -1,9 +1,64 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Row, Col } from 'antd';
+import fetch from 'isomorphic-unfetch';
+import Layout from '../components/layout';
+import './admin/profile/index.scss';
 
-const user = () => (
-    <div>
-        <h1> profile page</h1>
-    </div>
-);
+class User extends Component {
+    static async getInitialProps({ req }) {
+        let isLoggedIn = false;
+        let userData = {};
+        try {
+            const headers = req
+                ? {
+                    cookie: req.headers.cookie,
+				  }
+                : undefined;
+            const res = await fetch('http://localhost:5000/api/users/users', {
+                headers,
+            });
 
-export default user;
+            const userRes = await res.json();
+            console.log(userRes);
+            const { data } = userRes;
+            const { user } = data;
+            isLoggedIn = true;
+            userData = user;
+            console.log(`Show user fetched. : ${userData}`);
+        } catch (err) {
+            console.log(err);
+        }
+        return { isLoggedIn, userData };
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoggedIn: props.isLoggedIn,
+            user: props.userData,
+        };
```

Comment:
do not set state with items from props. Just use the `isLoggedIn` and `userData` where you need them in the component

Reply:
I think this is an example page he set so I know how to call the api. It isn't part of our project